tests/test_server.py

```diff
@@ -12,7 +12,7 @@ def test_server_download(tmpdir):
     download_model(
         url='https://docarray.jina.ai/_static/favicon.png',
         target_folder=tmpdir,
-        md5sum='a084999188f4290e2654aec43207ff2e',
+        md5sum='66ea4817d73514888dcf6c7d2b00016d',
```

Comment:
is there any method we can auto-calculate the `md5` in order not to block by this image chaning?

Reply:
no

Reply:
https://github.com/jina-ai/clip-as-service/blob/0b293ec834e80f7335aa625d683904594373a607/server/clip_server/model/pretrained_models.py#L124